.github/workflows/cherryPick.yml

```
@@ -151,6 +152,14 @@ jobs:
             Engineering
             Hourly
 
+      - name: Assign the PR to the deployer
+        if: ${{ steps.cherryPick.outputs.SHOULD_AUTOMERGE == 'false' || steps.isPullRequestMergeable.outputs.IS_MERGEABLE == 'false' }}
```

This is a recent change, but going forward we want to explicitly decode json variables in GitHub actions instead of relying on string comparisons:

```suggestion
        if: ${{ !fromJSON(steps.cherryPick.outputs.SHOULD_AUTOMERGE) || !fromJSON(steps.isPullRequestMergeable.outputs.IS_MERGEABLE) }}
```